addons/bus/static/src/im_status_service.js

```diff
@@ -41,9 +42,16 @@ export const imStatusService = {
             }
         };
 
-        bus_service.addEventListener("connect", () => {
-            browser.setTimeout(updateBusPresence, FIRST_UPDATE_DELAY);
-        });
+        bus_service.addEventListener(
+            "subscribed",
```

Comment:
Just to make sure, do always trigger `_updateChannels` initially when we need?

There's at least condition on `force || shouldUpdateChannelSubscription`.